src/App.jsx

```diff
@@ -146,6 +148,15 @@ export default function App() {
             </span>
           </div>
           <div>
+            <Button
+              onClick={() => setIsResetModalOpen(!isResetModalOpen)}
+              label="Reset"
+            />
+            <Modal
+              isOpen={isResetModalOpen}
+              onCancel={() => setIsResetModalOpen(false)}
+              onConfirm={() => { setMyDeck([]); setIsResetModalOpen(false); }}
```

Comment:
```suggestion
              onConfirm={() => {
                setMyDeck([]);
                setIsResetModalOpen(false);
              }}
```
No need for this to be a one liner.